README.md

```diff
@@ -66,6 +78,7 @@ How does module.exports help us with separation of concerns?
 
 ```text
 
+A module takes a bunch related functions and saves it into a package of code that can be used in another file. This helps a lot with separations of concerns because you can create long functions in one file, export them with modules, and then require them in another file.
```

Comment:
:+1:  You also get to control the interface - control what is available to the other file.